tools/profiling/python/pbt2ptt.pyx

```diff
@@ -44,7 +44,8 @@ import traceback
 from parsec_trace_tables import * # the pure Python classes
 from common_utils import *
 
-multiprocess_io_cap = 9 # this seems to be a good default on ICL machines
+#multiprocess_io_cap = 9 # this seems to be a good default on ICL machines
+multiprocess_io_cap = 128
```

Comment:
Whoops, this shouldn't have been included—it will be removed in the next iteration I push. I just modified this for better performance on the machines where I do the analysis.